spec/classes/plugin/default_hostgroup_spec.rb

```diff
@@ -2,4 +2,38 @@
 
 describe 'foreman::plugin::default_hostgroup' do
   include_examples 'basic foreman plugin tests', 'default_hostgroup'
+
+  context 'with user provided config hash' do
+    let(:params) do
+      {
+        'Redhat':
+          'osfamily': 'RedHat'
+          'lsbdistcodename': 'Santiago'
+        'Osx/common':
+          'osfamily': 'Darwin'
+        'Base':
+          'hostname': '.*'
+      }
+
+hostgroups = <<EOF
```

Comment:
You can indent this line. You can even indent the content using `<<~`.

Reply:
I was trying to execute rspec locally but did not find a way to do it. Can you share how to execute the spec test locally.

Reply:
The `~` removes the indentation and I am trying to test the exact content.

Reply:
https://github.com/theforeman/puppet-foreman/blob/master/CONTRIBUTING.md#getting-started has a bit more, but the tl;dr is:
```
bundle install
bundle exec rake spec_prep
bundle exec rspec spec/classes/plugin/default_hostgroup_spec.rb
```